### DISCUSSION TITLE
Could not load type 'RestSharp.IRestClient' from assembly 'RestSharp, Version=108.0.1.0

### DISCUSSION THREAD

**Discussion by jaylee71** [abuzuhri/Amazon-SP-API-CSharp#236] [Q&A]
Hello

Trying out the sample code and I am getting "Could not load type 'RestSharp.IRestClient' from assembly 'RestSharp, Version=108.0.1.0, Culture=neutral, PublicKeyToken=598062e77f915f75'." error when trying to make any type of call. Latest versions of RestSharp and FIKA is installed. What am I doing wrong?

Thank for any help.

Jay

**Comment by abuzuhri** [comment]
First start test in new fresh project I’m thinking there problem with old version exist is same solution you try to use .

**Reply by jaylee71** [reply]
Weird. It does work when I create a new project.

I downgraded to 106.15 and it works within my existing project. Anything higher and it doesn't work. This will have to do for now. 

Anyway, thank you very much for the project. It has made everything so much easier.

**Reply by jaylee71** [reply]
It seems RestSharp.IRestClient has been deprecated. Can this be causing my issue?

https://restsharp.dev/v107/#deprecated-interfaces
